### PR TITLE
Added support for getting payments without shipping method.

### PR DESCRIPTION
**Pull request by alfredsgenkins**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
             "email": "ilja@scandiweb.com"
         }
     ],
+    "version": "2.6.0",
```
Comment (eli-l) [inline composer.json]: do not commit version into composer.json

File: src/Model/Resolver/GetCartForCustomer.php [hunk 2]
```diff
@@ -120,7 +121,8 @@ public function resolve(
             [
                 'items' => $itemsData,
                 'tax_amount' => $tax_amount,
-                'discount_amount' => $discount_amount
+                'discount_amount' => $discount_amount,
+                'is_virtual' => !!$cart->getIsVirtual()
```
Comment (eli-l) [inline src/Model/Resolver/GetCartForCustomer.php]: According to PHPDoc getIsVirtual is returning bool, however in sources I can see it is coercing value to int. Worth to leave a comment for this coercion, let's prefer (bool) $variable over !!$variable

File: src/etc/schema.graphqls [hunk 3]
```diff
@@ -127,10 +125,13 @@ type PaymentMethod {
...
-type PaymentTotals implements TotalsObject {
-}
-
 interface TotalsObject @typeResolver(class: "\\ScandiPWA\\QuoteGraphQl\\Model\\Resolver\\TotalsObject") {
+    items_count: Int
+    customer_is_guest: Boolean
+    store_id: Int
+    customer_tax_class_id: Int
+    items_qty: Float
+    coupon_code: String
```
Comment (eli-l) [inline src/etc/schema.graphqls]: Additional improvements? Are these related to the feature?

File: src/etc/schema.graphqls [hunk 4]
```diff
@@ -104,13 +105,10 @@ input SaveAddressInformation {
...
 type QuoteData implements TotalsObject  {
-    id: Int
-    items_count: Int
-    customer_is_guest: Boolean
-    store_id: Int
-    customer_tax_class_id: Int
-    items_qty: Float
-    coupon_code: String
+    is_virtual: Boolean
+}
```
Comment (eli-l) [inline src/etc/schema.graphqls]: Is this 100% necessary? It might introduce breaking changes.

File: composer.json (hunk 1 above)
Comment (alfredsgenkins) [reply composer.json]: Thanks! I will remove.

File: src/etc/schema.graphqls (hunk 4 above)
Comment (alfredsgenkins) [reply src/etc/schema.graphqls]: This will not, this just exposes properties of the same totals object. Remember the issue when we wanted to use the totals returned from checkout in cart? This allows for full comparability.

File: src/etc/schema.graphqls (hunk 3 above)
Comment (alfredsgenkins) [reply src/etc/schema.graphqls]: Is this related to previous comment?

Another note: I have just moved fields from child to interface. It will still work. Only custom field like `is_virtual` is left in the interface.

File: src/Model/Resolver/GetCartForCustomer.php (hunk 2 above)
Comment (alfredsgenkins) [reply src/Model/Resolver/GetCartForCustomer.php]: Sure!
